### PR TITLE
Try fixing the extra empty line between fcs in html report

### PR DESCRIPTION
**Pull request by chuan-wang**

### CODE REVIEW

**Comment by chuan-wang** [comment]
Hi @aanil, the lane information in the `lanes_info.txt` file has no problem, but in `project_summary.html` there is an empty line between different FCs. Therefore I conclude that nothing is wrong with the data structure, but something might be wrong with the html template. In this PR the latest change made by Sofia was reverted to see whether the issue can be solved.

The change of `sorted_project_fcs` was independent of the issue and was purely for improvement.

File: data/report_templates/project_summary.md [hunk 1]
```diff
@@ -132,10 +132,10 @@ No lanes information to be displayed.
...
 Date | Flowcell | Lane | Clusters(M) | >=Q30(%) | PhiX | Method
 :-----|:----------|:------|:-------------|:----------|:------|:-------
-{%- for fc in project.flowcells.values()|sort(attribute='date') %}
-{%- for lane in fc.lanes.values() %}
+{% for fc in project.flowcells.values()|sort(attribute='date') -%}
+{% for lane in fc.lanes.values() -%}
 {{ fc.date }} | `{{ fc.name }}` | {{ lane.id }} | {{ lane.total_reads_proj }} | {{ lane.weighted_avg_qval_proj }} | {{ lane.phix }} | Seq. {{ fc.seq_meth }}
-{%- endfor %}
+{% endfor -%}
 {% endfor %}
```
Comment (aanil) [inline data/report_templates/project_summary.md:139]: Looking at old code as you said, the last endfor should probably be
```suggestion
{% endfor -%}
{%- endfor %}
```

**Review by aanil — APPROVED** [review]
👍

File: data/report_templates/project_summary.md (hunk 1 above)
Comment (aanil) [reply data/report_templates/project_summary.md:139]: I've made the changes on the latest stage if you want to check (including the last endfor)

**Review by aanil — APPROVED** [review]
👍

**Pull request merged**